src/GitLabApiClient/Models/Trees/Requests/TreeQueryOptions.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GitLabApiClient.Models.Trees.Requests
+{
+    public sealed class TreeQueryOptions
+    {
+        public string Reference { get; set; }
+        public string Path { get; set; }
+        public bool Recursive { get; set; }
+
+        internal TreeQueryOptions(string reference = null, string path = null, bool recursive = true)
```

Comment:
Recursive should not default to true
```suggestion
        internal TreeQueryOptions(string reference = null, string path = null, bool recursive = false)
```